host/libvirt_lxc_backend.go

```diff
@@ -669,11 +672,28 @@ func (c *libvirtContainer) watch(ready chan<- error) error {
 			g.Log(grohl.Data{"at": "get_streams", "status": "error", "err": err.Error()})
 			return err
 		}
+
 		log := c.l.openLog(c.job.ID)
 		defer log.Close()
-		// TODO: log errors from these
-		go log.Follow(1, stdout)
-		go log.Follow(2, stderr)
+
+		muxConfig := logmux.Config{
+			AppName: c.job.Metadata["flynn-controller.app"],
```

Comment:
ok, to clarify, here is the current format for syslog messages:

`<1>0 {timestamp} {hostID} {appID} {jobType}.{jobID} ID{fd} - {message}`

here is a sample of a message from a CI run:

`<0>1 2015-03-12T19:46:42.003397093Z flynn 68f43e1007584aba8d29b26286178aca printer.9ec352554d864355ae3835d645d26a56 ID1 - I like to print`

@titanous et al, does this look correct?